### DISCUSSION TITLE
冲刺回顾会议

### DISCUSSION THREAD

**Discussion by Zhishui02** [sansanchuiliu/ParkinglotManageSystem#15] [General]
待定

**Comment by zzh020326** [comment]
项目名称：停车场管理系统
会议日期：2024年10月30日
参会人员：
产品负责人（Product Owner）：张羽辰、熊志坤
开发团队（前端开发、后端开发、测试人员）：张俊辉、张治华、费亮
敏捷教练（Scrum Master）：田晋坤
1.会议目的
本次会议成功评审了第一次 Sprint 的完成情况，获得了利益相关者的反馈，并制定了明确的改进计划和行动项。希望在下一个 Sprint 中能够继续推进停车场管理系统的各项功能，实现更好的用户体验和管理效率。
2. Sprint 回顾讨论
2.1 做得好的方面
需求分析清晰：在会议中，能够明确各种用户和利益相关者的需求，确保系统设计能够满足实际需求。
技术方案讨论：针对系统的技术架构、数据库设计、接口设计等方面进行深入讨论，确保技术方案的可行性和扩展性。
用户体验设计：充分考虑用户体验，讨论停车场管理系统的界面设计、人机交互等，以提升用户的使用便利性。
数据安全和隐私保护：针对用户数据的安全性和隐私保护进行充分讨论，制定相应的安全策略。
培训与支持：讨论如何为停车场的工作人员提供培训和支持，确保他们能有效使用管理系统。
预算和资源分配：合理讨论预算，确保项目在资金和资源上的有效分配。
项目时间表：设定明确的项目进度和阶段性目标，确保项目按期推进。
风险管理：识别潜在的项目风险，制定应急预案和应对措施。
持续沟通机制：建立会议后的沟通机制，确保团队成员之间信息畅通。
反馈机制：讨论如何在系统上线后收集用户反馈，以便及时进行改进。
2.2需要改进的方面
产品负责人张羽辰指出，由于接口文档准备不足，部分与后端对接的任务--数据统计与报表功能被推迟到了下一个 Sprint。这是本次 Sprint 中未完成的部分，团队将在接下来的开发中优先处理。
1）增加车牌识别功能：开发团队将在下一个 Sprint 中实现自动识别进入和离开停车场的车辆车牌号码，以便快速准确地记录车辆的进出时间
2）部署视频监控系统：开发团队将在下一个 Sprint 中安装多台摄像头，以实现对停车场的全方位视频监控。摄像头将实时录制停车场的状况，并将视频数据存储在本地服务器或云端，便于查询和回放。例如，在发生车辆刮擦、盗窃等事件时，管理员可以通过查看监控视频来确认事件经过和责任方。
3）优化报警系统：开发团队将在下一个 Sprint 中完善报警系统。当系统检测到异常情况（如非法闯入、车位被占用超过规定时间等）时，将自动触发报警。报警方式包括声音警报、短信通知管理员，或推送通知至相关工作人员的手机应用上，以便及时处理问题。
3.改进措施
在下一次 Sprint 开始前，产品负责人和后端开发团队将加强接口文档的准备。计划在 Sprint 开始前进行接口文档的审查，确保所有接口都经过详细设计并记录在案，以避免开发过程中反复沟通。
